docs/tutorial/parameter-types/enum.md

```diff
@@ -66,3 +66,32 @@ Training neural network of type: lstm
 ```
 
 </div>
+
+
+### Using Enum names instead of values
+
+Some times you want to accept `Enum` names from command line and convert 
+that into `Enum` values in command handler. You can enable this with 
+`names=True` parameter:
+
+```Python hl_lines="14"
+{!../docs_src/parameter_types/enum/tutorial003.py!}
+```
+
+And then the names of the `Enum` will be used instead of values:
+
+<div class="termy">
+
+```console
+$ python main.py --log-level debug
+
+Log level set to DEBUG
+```
+
+</div>
+
+If `IntEnum` type is given, then enum names are used implicitly.
```

Comment:
I think this is reasonable, but I'm not sure about doing it implicitly. I'd rather go for consistency here.

Imagine a user who sees an `IntEnum` before seeing a str enum. They'll (reasonably) assume that enums use names instead of values by default. And then they'll be in for a surprise when they encounter the default behavior of all other enums.

Keeping it consistent seems best to me. Does anyone else have any thoughts on this?

Reply:
I agree - consistency is more important here.